pkg/controller/apiserver/apiserver_controller.go

```diff
@@ -252,11 +252,7 @@ func (r *ReconcileAPIServer) Reconcile(ctx context.Context, request reconcile.Re
 		r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying installation", err, reqLogger)
 		return reconcile.Result{}, err
 	}
-	if variant == "" {
```

Comment:
@tmjd @gcosgrave I think we still do need this check to handle the tiniest of race conditions.

If a user creates an Installation with no variant, we want to wait for the Installation controller to default it before we continue. It should only add a second of delay.

Reply:
Yeah I agree. To be specific the check can be changed to `if installationSpec.Variant == ""`

Reply:
done